Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM rust:1.65.0-buster
+RUN apt-get update && \
+    apt-get install -y g++ cmake llvm-dev clang libclang-dev
+WORKDIR /usr/src/myapp
+COPY . .
```

Comment:
Can we not build the iowatcher inside the container? That way we avoid needed all the build tools installed on the host machine?

Reply:
That's exactly what that Dockerfile does. We would improve it with multistage build, but yep, it builds within the container.

Reply:
Fixed in multistage build.